### ISSUE TITLE
JSONB columns display as [object Object]

### ISSUE THREAD

**Issue by k3rni** [beekeeper-studio/beekeeper-studio#57]
![image](https://user-images.githubusercontent.com/200996/81264702-24ed6380-9042-11ea-89f3-b7792b8b77b9.png)

here, the `data` column is JSONB with a toplevel array of objects.

Same for query output.

```
$ snap list beekeeper-studio
Name              Version  Rev  Tracking       Publisher         Notes
beekeeper-studio  1.2.0    18   latest/stable  matthew-rathbone  -

**Comment by rathboma** [comment]
lol, what's the problem that looks soo good!

**Comment by rathboma** [comment]
This is fixed, see release 1.3 💪

**Comment by k3rni** [comment]
installed 1.3 with `snap refresh --edge`, fixed!
